src/components/views/messages/SenderProfile.tsx

```diff
@@ -116,7 +117,10 @@ export default class SenderProfile extends React.Component<IProps, IState> {
         if (disambiguate) {
             mxidElement = (
                 <span className="mx_SenderProfile_mxid">
-                    { mxid }
+                    { UserIdentifier.getDisplayUserIdentifier(
+                        mxid, { withDisplayName: true, roomId: mxEvent.getRoomId() },
+                    )
+                    }
```

Comment:
Not sure of great way to format this, but maybe:

```suggestion
                    ) }
```